test/hotspot/jtreg/serviceability/jvmti/vthread/StopThreadTest/StopThreadTest.java

```diff
@@ -248,6 +240,7 @@ public void run() {
         //  - when suspended: JVMTI_ERROR_NONE is expected
         static void A() {
             log("TestTask.A: started");
+            atPointA = true;
```

Comment:
What happens if we are at the point where this flag has been set true, but we have not yet executed the monitorenter? Will the test pass if the StopThread is done a bit early here?

Reply:
Yes, the test will pass.
#A.1: The `StopThread` should still return the JVMTI_ERROR_THREAD_NOT_SUSPENDED error code.
#A.2: The `StopThread` should return the JVMTI_ERROR_NONE as the target virtual thread has been mounted.